File: 3731-find-missing-elements/3731-find-missing-elements.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findMissingElements = function(nums) {
    const ans=[]
    for(let i=Math.min(...nums);i<Math.max(...nums);i++){
        if(!nums.includes(i)){ans.push(i)}
    }
    if (ans) return ans
    else return []
};

File: 3731-find-missing-elements/3731-find-missing-elements.py
class Solution(object):
    def findMissingElements(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        lst=[]
        i=min(nums)
        large=max(nums)
        while i<large:
            if i not  in nums:
                lst.append(i)
            i+=1
        if lst:
            return lst
        else:
            return []
        
        